tests/Integration/ScriptsTest.php

```diff
@@ -144,15 +137,14 @@ public function test_enqueue_js_tracker(): void {
 	 * @uses \Parsely\Parsely::update_metadata_endpoint
 	 * @uses \Parsely\Scripts::register_scripts
 	 * @uses \Parsely\Scripts::script_loader_tag
+	 * @group enqueue-js
 	 * @group insert-js
 	 */
 	public function test_enqueue_js_tracker_with_cloudflare(): void {
```

Comment:
This test method isn't checking the enqueueing, but the correct building of the output markup based on a filter value. As such, it seems like this test (the only one left with output buffering) could be moved out of the middle of the other tests that check for correct enqueueing, and the method renamed to indicate it's more about the markup. This test is checking two lines of https://github.com/Parsely/wp-parsely/blob/b9cf935255c63e707ff5b86dc0ecb9d142ab1e46/src/class-scripts.php#L172.

Reply:
Correct, I will do this. We could also move it to another file which would be dedicated to any output-specific tests.

Reply:
This has not been marked as resolved yet, so I'm not sure if there's anything additional to do. I have updated the `@group` designation and have also moved the function towards the bottom.